html/doc/build_ngx_pagespeed_from_source.html

```diff
@@ -137,7 +137,10 @@
 If you're using dynamic modules to integrate with an already-built nginx, make
 sure you pass <code>./configure</code> the same arguments you gave it when
 building nginx the first time.  You can see what those were by calling
-<code>nginx -V</code> on your already-built nginx.
+<code>nginx -V</code> on your already-built nginx. (Note: Nginx installations provided
+by package managers have been observed to additionally need
```

Comment:
@morlovich re : "If you are aware of particular ones, may be worth mentioning them?"
done